ui/DialogService.ts

```
@@ -0,0 +1,38 @@
+  SUCCESS,
+  CONFIRMATION,
+}
+
+class DialogService {
+  show(title: string, message: string, type: DialogType): void {
+    switch (type) {
+      case DialogType.INFO:
+        alert(`${title}\n\n${message}`);
+        break;
+      case DialogType.CONFIRMATION:
+        const result = confirm(`${title}\n\n${message}`);
```

Biome (2.4.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)